core/tests/test_utils.py

```diff
@@ -2241,10 +2241,11 @@ def get_json(self, url, params=None, expected_status_int=200):
         return self._parse_json_response(json_response, expect_errors)
 
     def post_json(
-            self, url, payload, csrf_token=None, expected_status_int=200,
-            upload_files=None):
+            self, url, data, csrf_token=None, expected_status_int=200,
+            upload_files=None, use_payload=True):
         """Post an object to the server by JSON; return the received object."""
```

Comment:
Add Args docstring (in particular to explain what use_payload represents).

Reply:
Done